### DISCUSSION TITLE
Downlight not recognized. New Firmware DP 41

### DISCUSSION THREAD

**Discussion by Xatinni** [make-all/tuya-local#1610] [Q&A]
I'm trying to add a new Arlec Smart LED Downlight, I already have 13 installed without issue, I can't get two of the new ones to work.  I believe the issue is related to new firmware that supports a new datapoint 41 - Wi-Fi Remote Control Switch, I've tried adding it with the standard dp's for this device and with/without 41, there is actually nowhere to configure 41 on the form if I include it. I can control the light from both the tuya app and API explorer (Tuya Dev), just doesn't work in local Tuya, any help would be greatly appreciated. 
20 on/off
21 Color mode
22 Brightness
23 Color Temp
24 Color
25 Scene
41 Wi-Fi Remote Control Switch

FYI all the other lights were auto discovered, these two weren't, so am trying to manually configure, could be user error on my part. local key and IP are correct eg checked mac in app and ip/mac in router and queried key using Tuya dev device details endpoint. I haven't updated my working lights to the new firmware, happy to test but not sure how to restore the original firmware if it breaks.

![Tuya](https://github.com/make-all/tuya-local/assets/5786215/9340eff5-8865-4e4d-b1c6-09d8eaf8a78c)

Link to device
https://www.bunnings.com.au/arlec-9w-720lm-rgb-cct-grid-connect-smart-led-downlight_p0335074

New Feature in updated firmware
https://grid-connect.com.au/smart-home-products/wireless-sync/

**Comment by make-all** [comment]
The new firmware might be protocol version 3.5, which does not work well with protocol auto detection.

**Reply by make-all** [reply]
Alternatively, the devices may require that dp 41 to be set to true through either cloud or bluetooth before they can be locally controlled via wifi, and the setting may have a timeout where it auto-resets to false.

**Reply by Xatinni** [reply]
Thanks for the guidance @make-all, a couple of things you prompted, always make sure you are on the latest version before making a post... embarrassed, was on 5.1..   didn't solve the issue, but the protocol version comment made me do some more research... thanks again. can confirm that the two devices in question are protocol 3.5... yay.. so thinking I just park these until 3.5 is supported. where do I send the beer..

![tuya2](https://github.com/make-all/tuya-local/assets/5786215/ee40b60e-900d-40fe-9c86-048bf8bbc390)

**Reply by make-all** [reply]
3.5 is supported. What is 5.1?

**Reply by Xatinni** [reply]
humble pie, im running the one through HACs which is mantained by [rospogrigio](https://github.com/rospogrigio)
/
[localtuya](https://github.com/rospogrigio/localtuya)
